scripts/upgrade

```diff
@@ -9,110 +9,36 @@
 source _common.sh
 source /usr/share/yunohost/helpers
 
-#=================================================
-# MANAGE SCRIPT FAILURE
-#=================================================
-
-ynh_clean_setup () {
-  ynh_clean_check_starting
-}
-# Exit if an error occurs during the execution of the script
-ynh_abort_if_errors
-
 #=================================================
 # LOAD SETTINGS
 #=================================================
 ynh_script_progression --message="Loading installation settings..." --weight=2
 
-app=$YNH_APP_INSTANCE_NAME
-
-domain=$(ynh_app_setting_get --app=$app --key=domain)
-path_url=$(ynh_app_setting_get --app=$app --key=path)
-port=$(ynh_app_setting_get --app=$app --key=port)
-final_path=$(ynh_app_setting_get --app=$app --key=final_path)
-language=$(ynh_app_setting_get --app=$app --key=language)
-version=$(ynh_app_setting_get --app=$app --key=version)
-data_path=$(ynh_app_setting_get --app=$app --key=data_path)
 team_display_name=$(ynh_app_setting_get --app=$app --key=team_display_name)
-admin=$(ynh_app_setting_get --app=$app --key=admin)
-password=$(ynh_app_setting_get --app=$app --key=password)
 email=$(ynh_user_get_info --username=$admin --key=mail)
-db_name=$(ynh_sanitize_dbid --db_name=$app)
-db_user=$db_name
 
 #=================================================
 # CHECK VERSION
 #=================================================
-ynh_script_progression --message="Checking version..." --weight=1
 
-previous_upstream_version="$(ynh_app_upstream_version --manifest="/etc/yunohost/apps/$YNH_APP_INSTANCE_NAME/manifest.json")"
 upgrade_type=$(ynh_check_app_version_changed)
 
-#=================================================
-# BACKUP BEFORE UPGRADE THEN ACTIVE TRAP
-#=================================================
-ynh_script_progression --message="Backing up the app before upgrading (may take a while)..." --weight=1
-
-# Backup the current version of the app
-ynh_backup_before_upgrade
-ynh_clean_setup () {
-  # Remove the PostgreSQL db in case of error during the migration.
-  if [ -n "$remove_psql_in_case_of_error" ]; then
```

Comment:
If this code path is removed, the `    remove_psql_in_case_of_error=1` on line 51 becomes useless. And yet we want a way to delete the PG database in case of a failed migration.